e2e/ansible/playbooks/feature/snapshots/percona-mysql/mysql-snapshot-prerequisites.yml

```diff
@@ -1,12 +1,4 @@
 ---
```

Comment:
Let's add the equivalent of the apt update ansible module with shell module w/ ignore errors (skip lint)